### ISSUE TITLE
Enable add-on button has no shortcut key in NVDA Add-ons manager

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#6592]
In the Add-ons manager (Press NVDA+N, then choose "Tools", then "Manage Add-ons"), the "Enable add-on" button does not have a shortcut key.  Every other button has an underlined letter, but not Enable Add-ons (it's possible it does but not labelled?)  My suggestion would be alt+e.

The button is still accessible with TAB.

CC @feerrenrut

**Comment by josephsl** [comment]
Fixed as part of #6388. Thanks.

**Comment by Qchristensen** [comment]
Thanks Joseph,

Sorry I was using 2016.3 at the time and didn't think to check master.  I
confirm it works in current master build.


-- 
22 Point

Web: http://www.22point.com.au
Facebook: https://www.facebook.com/22Point
Twitter: https://twitter.com/22PointApps

Check out our first app, RapiTap! - Tap targets fast & avoid decoys:
Adrenaline pumping, Challenging, Accessible!
https://play.google.com/store/apps/details?id=au.com.twentytwopoint.rapitap
Free trial:
https://play.google.com/store/apps/details?id=au.com.twentytwopoint.rapitapfree
